--- a/bigtop-deploy/puppet/modules/hadoop/manifests/init.pp.orig        2016-04-17 01:17:45.311326664 +0000
+++ b/bigtop-deploy/puppet/modules/hadoop/manifests/init.pp     2016-04-17 01:38:12.703326664 +0000
@@ -61,11 +61,9 @@ class hadoop ($hadoop_security_authentic
       include hadoop::historyserver
       include hadoop::proxyserver

-      Class['Hadoop::Init_hdfs'] -> Class['Hadoop::Resourcemanager']
       if ("nodemanager" in $roles) {
         Class['Hadoop::Resourcemanager'] -> Class['Hadoop::Nodemanager']
       }
-      Class['Hadoop::Init_hdfs'] -> Class['Hadoop::Historyserver']
     }

     if ($hadoop::common_hdfs::ha == "disabled" and "secondarynamenode" in $roles) {
